{{cookiecutter.project_name}}/docs/index.rst

```diff
@@ -1,5 +1,10 @@
-{{cookiecutter.project_name}}
-==============================
+Welcome to {{cookiecutter.project_name}}!
+=========================================
+
+Welcome to the tryal library.
```

Comment:
instead of tryle put the template name

pls als add the project name to spelling_wordlist